packages/core/src/page.ts

```
@@ -0,0 +1,173 @@
+import { readFileSync } from 'node:fs'
+import { extname } from 'node:path'
+import { normalizePath } from 'vite'
+import { assign as cjAssign } from 'comment-json'
```

ESLint

[error] Expected "comment-json" to come before "vite".

(perfectionist/sort-imports)